packages/components/src/Spinner/src/Spinner.module.css

```diff
@@ -55,8 +55,8 @@
     grid-area: spinner;
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/ee73cb2c-3318-412f-906f-5eaf314157fc)
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/bacfd6ea-9052-43de-ac66-bf0eeb05b00f)

I haven't found how but in some situation the trigger does not have the same look when opened.

Reply:
This happens in light mode too.